go.mod

```diff
@@ -77,7 +77,7 @@ require (
 	github.com/invopop/jsonschema v0.7.0
 	github.com/jackc/pgx/v5 v5.5.4
 	github.com/muesli/reflow v0.3.0
-	github.com/openvex/go-vex v0.2.5
+	github.com/openvex/go-vex v0.2.6-0.20240506020847-46399d425c68
```

Comment:
I updated the library manually. The reason behind it is that in recent commits there are a bunch of types that are implemented which are useful since we don't have to implement them ourselves.

AFAIK, no cross-reference more than the base CSAF document which them inherit the rest of the profiles from the specification.

Reply:
but openvex is not CSAF vex is it? Or what am I missing?

Reply:
Yeah, I see, I replied below, this openvex library seems to have some types for csaf.